source/adapter.js

```diff
@@ -254,13 +254,71 @@ AdapterJS.maybeFixConfiguration = function (pcConfig) {
 AdapterJS.addEvent = function(elem, evnt, func) {
   if (elem.addEventListener) { // W3C DOM
     elem.addEventListener(evnt, func, false);
-  } else if (elem.attachEvent) {// OLD IE DOM 
+  } else if (elem.attachEvent) {// OLD IE DOM
     elem.attachEvent('on'+evnt, func);
   } else { // No much to do
     elem[evnt] = func;
   }
 };
 
+AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink) {
```

Comment:
@johache I wanted to re-use to prompt users to install the chrome web extension so I moved it there. But in the end we redirected users to the web store instead. If you prefer it in the plugin scope I can move it.